.github/workflows/ci.yml

```diff
@@ -36,17 +35,29 @@ jobs:
       uses: peaceiris/actions-mdbook@v1
       with:
         mdbook-version: latest
-    - name: Install C libraries for tooling
+    - name: Install C libraries for tooling on ubuntu
       if: matrix.os == 'ubuntu-latest'
       run: sudo apt-get update && sudo apt-get install libudev-dev libusb-1.0-0-dev
+    - name: install C libraries for tooling on macOS
+      if: matrix.os == 'macOS-latest'
+      run: brew install libusb
     - name: Build
       run: RUSTFLAGS='--deny warnings' cargo build
```

Comment:
Oh, I didn't notice this, that explains why it failed on Windows. The variable can be moved up to the `env:` key, then `shell: bash` won't be necessary (and we catch more warnings).

Reply:
we can't move it to the `env:` key, because that's what broke our qemu runs so we moved it: https://github.com/knurling-rs/defmt/pull/27/files

Reply:
Oh, duh, that's because Cargo still doesn't merge the variable with `.cargo/config`, right